### DISCUSSION TITLE
Huoltoikkuna 13.9.

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#766] [Huolto- ja häiriötiedotteet]
13.9. Huoltoikkunassa bootataan kaikki palvelimet, jotta uusi tiedostojen maksimimääräasetus saadaan voimaan ja nextit voidaan käynnistellä takaisin versiopäivityksen valmistelua varten. Tästä aiheutuu kaikissa kimpoissa 13.9. aamulla lyhyt, ehkä noin 10 minuutin käyttökatko.

**Comment by CodoDerDritte** [comment]
Huollot alkavat ja testit ovat nyt alhaalla. Alhaalla käyvät seuraavaksi TäTi, Siili ja Lumme.

**Comment by CodoDerDritte** [comment]
Meg käynnistyy uudelleen ja kunhan se on takaisin koivillaan, niin käynnistellään nuo TäTi, Siili ja Lumme takaisin. Seuraavaksi alas menevät Kirkes, Kyyti ja Vaara.

**Comment by CodoDerDritte** [comment]
Täti, Lumme ja Siili on käynnistetty takaisin, seuraavaksi alas menevät Helle ja Outi. Kirkes, Kyyti ja Vaara käynnistetään takaisin kunhan kakkosnode (Jo) ehtii käynnistyä uudelleen.

**Comment by CodoDerDritte** [comment]
Kirkes, Kyyti ja Vaara on käynnistetty takaisin. Helle ja Outi palaavat linjoille kunhan Node 3 (Beth) ehtii bootata. Seuraavaksi sitten alas Lappi ja Vaski.

**Reply by Kati-Helle** [reply]
Kiitos. Helle nyt hereillä.

**Comment by CodoDerDritte** [comment]
Juu, Helle ja Outi tosiaan nyt taas käynnissä. Nelosnoden (Amy) boottia odotellaan vielä ja sen jälkeen sitten käynnistellään Lappi ja Vaski.

**Comment by CodoDerDritte** [comment]
Lappi ja Vaskikin nyt taas linjoilla. Käynnistellään testit ja nextit tässä aamun aikana, mutta tuotantojen osalta huoltokatko päättyy tähän. Muistattehan että hyllyvarausraportti päivittyy vasta katkon "virallisen" ajan jälkeen, eli kello yhdeksän.

**Comment by CodoDerDritte** [comment]
Testit on nyt herätelty henkiin.

**Comment by CodoDerDritte** [comment]
Nextit nyt myös käynnissä. Tältä keskiviikolta homma oli tässä. Huolto päättyy.
